.python-version

```diff
@@ -0,0 +1 @@
+3.13
```

Comment:
I don't love that this file exists, to be honest, but when uv is run in "project mode" it emits a warning if it's not here.

Reply:
Maybe we can just have the workflow generate it so that we don't have it in the repo?

Reply:
Intriguing. Could rip it right out of the similar line in `pyproject.toml`. Would reduce the need to manually update in the future as well.

Reply:
Okay, done. Now that I'm thinking about it, the build process could run the uv installer if there wasn't already an uv, but maybe we're getting a little too fancy.